pynicotine/gtkgui/search.py

```diff
@@ -612,39 +612,23 @@ def update_filter_widgets(self):
             self.clear_undo_filters_button.set_tooltip_text(tooltip_text)
             self.clear_undo_filters_icon.set_property("icon-name", icon_name)
 
-    def populate_filters(self):
+    def populate_default_filters(self):
 
         if not config.sections["searches"]["enablefilters"]:
             return
 
         sfilter = config.sections["searches"]["defilter"]
         num_filters = len(sfilter)
-        stored_filters = {}
+        stored_filters = self.FILTERS_EMPTY
```

Comment:
```suggestion
        stored_filters = self.FILTERS_EMPTY.copy()
```

otherwise you modify the constant